github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -572,6 +572,7 @@ presubmits:
       vgpu: "true"
     max_concurrency: 1
     name: pull-kubevirt-e2e-kind-1.23-vgpu
+    optional: true
```

Comment:
Should this be optional?

Reply:
No we want that to be required - will update.

Reply:
Should be updated now.